### PR TITLE
Fixing response headers

### PR DESCRIPTION
**Pull request by adrianovalente**
**This PR:** fixes issue #16

### CODE REVIEW

**Comment by adrianovalente** [comment]
@StefanoHellebrekers @lucasBgabriel @guilhermemamprin CR when  you can

File: init/express/init.js [hunk 1]
```diff
@@ -17,6 +17,19 @@ function init(cb) {
 
 function configureRoutes() {
   app.use(bodyParser.json());
+
+
+  app.all('*', function(req, res, next) {
+
+    console.log('Im here');
```
Comment (guilhermemamprin) [inline init/express/init.js]: I think this for debugging only! @adrianovalente

**Comment by guilhermemamprin** [comment]
:+1:

**Comment by lucasBgabriel** [comment]
👍

**Pull request merged**